guides/common/modules/con_converting-a-host-to-rhel.adoc

```diff
@@ -0,0 +1,76 @@
+Then you perform the actual conversion on the host using the Ansible playbook, which installs the Convert2RHEL CLI tool on the host and runs it.
+
+You can use the Ansible role to generate conversion data for the following conversions:
+
+* CentOS Linux 7 to {RHEL} 7
+* Oracle Linux 7 to {RHEL} 7
+* CentOS Linux 8 to {RHEL} 8
+* Oracle Linux 8 to {RHEL} 8
+
+These conversions are supported by Red Hat.
+ifndef::satellite[]
+You can use the Ansible playbook for other conversions as well, but in those cases you must enable required repositories and configure activation keys manually instead of using the Ansible role and variables.
+However, this approach was not tested and is not supported.
+endif::[]
+
+The conversion process is similar to a minor release upgrade of {RHEL} in which every RPM package on the system is replaced.
+Third-party packages and non-Red Hat packages that are not available in {RHEL} are retained.
+
+The Convert2RHEL utility removes unnecessary packages such as logos or packages known to cause issues during the conversion.
+The utility replaces the `CentOS-release` or `Oracle-release` package with the `rhel-release` package, and all packages signed by CentOS or Oracle with their Red Hat equivalents.
+The utility also subscribes the host to {Team} Subscription Management.
+
+The duration of the conversion process depends on the number of packages that have to be replaced, network speed, storage speed, and similar factors.
+
+.Prerequisites
+* Review https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html-single/converting_from_an_rpm-based_linux_distribution_to_rhel/index#con_supported-conversion-paths_converting-from-a-linux-distribution-to-rhel[Supported conversion paths] in _Converting from an RPM-based Linux distribution to RHEL_.
+* You must have completed the steps 1.{range}5. of the procedure https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html-single/converting_from_an_rpm-based_linux_distribution_to_rhel/index#proc_preparing-for-a-rhel-conversion_converting-from-a-linux-distribution-to-rhel[Preparing for a RHEL conversion] in _Converting from an RPM-based Linux distribution to RHEL_.
+* Ensure you have a subscription manifest uploaded to your {Project} and that there are sufficient {RHEL} entitlements allocated for the conversions you intend.
+Alternatively, you can use Ansible variables to tell the role to import the manifest from disk.
+The manifest must be imported to the organization to which you will register hosts for conversion.
++
+You can update your allocations and download the updated manifest from the https://access.redhat.com[Red Hat Customer Portal].
+ifndef::orcharhino[]
+For more information, see https://access.redhat.com/documentation/en-us/red_hat_subscription_management/2022/html/using_red_hat_subscription_management/using_manifests_con[Using manifests] in _Red Hat Subscription Management_.
+endif::[]
```

Comment:
@maximiliankolb I'm not sure if this should be included for orcharhino as well or not. Pls let me know.